### PR TITLE
Try another info layout, and fix the services tags to display page title

### PR DESCRIPTION
**Pull request by skavhaug**
This layout PR changes the order of the information element on the single view in about, and uses the page title instread of service tag.

### CODE REVIEW

File: layouts/about/single.html [hunk 1]
```diff
@@ -5,23 +5,8 @@ <h1 class="center">{{ .Title }}</h1>
...
-        {{- with $.Site.GetPage (printf "/%s/%s" $taxo $service) -}}
-            <a href="{{ .Permalink }}">{{ $service }}</a>
-        {{- end -}}
-    {{- end -}}
-{{ end }}
-</p>
-
-<ul>
+<div class="center">
+<ul style="list-style-type:none">
```
Comment (eigilhs) [inline layouts/about/single.html]: I would prefer to have this as a class in our css file. Inline styles hurt my eyes a little. But I like how the result looks!

Comment (skavhaug) [reply layouts/about/single.html]: Feel free to fix this, mr. css guru! Or I can do it by applying the trial and error method :)

**Comment by skavhaug** [comment]
I've done some more tweaks, and added the linkedin logo. Just a suggestion, but I think it looks ok.

**Comment by eigilhs** [comment]
I think the logo looks a bit too much like an ad for linkedin. Maybe we could use something simpler, like an icon next to the text?

**Comment by mandus** [comment]
I agree with Eigils last comment, the linkedin logo https://content.linkedin.com/content/dam/me/business/en-us/amp/brand-site/v2/bg/LI-Bug.svg.original.svg will look better, and then we can add Git{hub,lab}, and others..

**Comment by eigilhs** [comment]
Also I think they should all be in the font color, or at least the same color for all the icons.

**Comment by skavhaug** [comment]
Det er begrenset hva man får lov til å gjøre med slike logoer.

**Comment by eigilhs** [comment]
Can't see that it would be a problem. It's [pretty](https://www.kodemaker.no/folk/) [common](https://www.jpro.no/menneskene).

**Comment by skavhaug** [comment]
Have you read the conditions?

**Comment by eigilhs** [comment]
Yes. I still can't imagine it would be an issue giving the icons the same color as the font. The web is full of social media icon arrays in monochrome.

**Comment by skavhaug** [comment]
Ok, then let’s do it. Will fix

**Comment by eigilhs** [comment]
I especially take comfort in the fact that patentstyret.no does it in their footer.

**Comment by skavhaug** [comment]
Valid point. But they could be amateurs?

**Comment by skavhaug** [comment]
Can @mandus and @eigilhs pls have a look at this? Even though it might not be perfect, its an improvement from current state, and I would like to have it merged. The small logo is now in place, but I have not changed the color.

**Pull request merged**